unit-tests/live/streaming/test-jpeg-compressed-format.py

```diff
@@ -0,0 +1,40 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2025 Intel Corporation. All Rights Reserved.
+
+# test:device D555
+
+
+import pyrealsense2 as rs
+from rspy import test
+from rspy import log
+
+
+with test.closure( "Check jpeg format support"):
+    ctx = rs.context({"format-conversion":"raw"})
+    color_sensor = ctx.query_devices()[0].first_color_sensor()
+    jpeg_profile = None
+    for p in color_sensor.profiles:
+        if p.stream_type() == rs.stream.color and p.format() == rs.format.mjpeg:
+            jpeg_profile = p
+            break
+                
+if jpeg_profile:
+    log.d("Device supports jpeg streaming with profile:", jpeg_profile)
+else:
+    log.d("Device does not support jpeg streaming")
+    test.print_results_and_exit()
+    
+    
+    
+with test.closure("Check streaming and conversion to RGB8"):
+    pipeline = rs.pipeline()
+    config = rs.config()
+    vp = jpeg_profile.as_video_stream_profile()
+    config.enable_stream(rs.stream.color, vp.stream_index(), vp.width(), vp.height(), rs.format.rgb8, vp.fps()) # JPEG is converted to RGB8
+    pipeline.start(config)
+
+    for i in range(10):
+        frames = pipeline.wait_for_frames()
+        log.d( frames )    
```

Comment:
Not sure if I understand, how do we know we get jpeg that was converted to RGB and not YUYV that was converted to RGB?
Because of the stream index?

Reply:
How can we test that indeed the origin was Jpeg? any way?

Reply:
We don't have a way to verify frame original format. We currently base it on stream index as found by the previous test.

Reply:
If the camera does not support JPEG we will exit the test with success return value before trying to stream.
If you think it will be clearer I will merge the two tests into one with an if/else.

When the camera does support JPEG and there is a problem with the frames `wait_for_frames` will throw.